packages/atoms/src/Text.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Text as RNText, TextProps as RNTextProps, TextStyle } from 'react-native';
+import { useNativeStyles, prepareNativeStyle, NativeStyleObject } from '@trezor/styles';
+import { Color, TypographyStyle } from '@trezor/theme';
+
+interface TextProps extends Omit<RNTextProps, 'style'> {
```

Comment:
I would always keep component prop types right above components like we do now. And styled above those

Reply:
I don't see any benefit of doing that, and for me makes more sense to keep prop types always at top. It makes sense because in most cases when you open file you only want to look how you can use component and prop types are kind of docs for component. If you will put prop types above component it will be always in different place because of styles and and other stuff and you will have to find it.

Reply:
I enjoy when types are next to the function it's used it so everything is grouped. Yes, you have to scroll a bit to find the props but it also would be easy since it's visible where the component-related code starts :D And more often I find myself needing to edit props from within a component than going inside to read them, since it's often possible to preview them with code autocompletion. But again, can't be forcing this alone.

Reply:
I personally also have a slight preference to have component prop types right above component render method, but it doesn't seem important to me.  If you do, I suggest we vote on it in the team and put it in a future "Code style guide"...